AmplifyPlugins/API/Tests/APIHostApp/AWSAPIPluginGen2GraphQLTests/Gen2_4/Customer4+Schema.swift

```diff
@@ -32,7 +32,7 @@ extension Customer4 {
     model.fields(
       .field(customer4.id, is: .required, ofType: .string),
       .field(customer4.name, is: .optional, ofType: .string),
-      .hasOne(customer4.activeCart, is: .optional, ofType: Cart4.self, associatedWith: Cart4.keys.customer),
+      .hasOne(customer4.activeCart, is: .optional, ofType: Cart4.self, associatedFields: [Cart4.keys.customer]),
```

Comment:
This is the problem we saw that it failed to compile so I had manually updated to `associatedWith`. The codegenerated output is `associatedFields` array.